libzecale/circuits/fields/fp12_2over3over2_gadgets.hpp

```diff
@@ -193,9 +193,9 @@ class Fp12_2over3over2_mul_gadget
     Fp12_2over3over2_variable<Fp12T> _A;
     Fp12_2over3over2_variable<Fp12T> _B;
     Fp12_2over3over2_variable<Fp12T> _result;
-    Fp6_3over2_mul_gadget<Fp6T> _v0;
-    Fp6_3over2_mul_gadget<Fp6T> _v1;
-    Fp6_3over2_mul_gadget<Fp6T> _a0_plus_a1_times_b0_plus_b1;
+    Fp6_3over2_mul_gadget<Fp6T> _compute_v0;
+    Fp6_3over2_mul_gadget<Fp6T> _compute_v1;
+    Fp6_3over2_mul_gadget<Fp6T> _compute_a0_plus_a1_times_b0_plus_b1;
```

Comment:
A few lines above you have `_compute_z1z3z5_times_x0x2x4` and here you have `_compute_a0_plus_a1_times_b0_plus_b1`, which is not consistent

Reply:
renamed `_compute_z1z3z5_times_x0x2x4`.

Reply:
Thanks.
To be clear, `_compute_z1z3z5_times_x0x2x4` was fine by me, i.e. you could also have decided to rename `_compute_a0_plus_a1_times_b0_plus_b1` as `_compute_a0a1_times_b0b1` to be consistent.